langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -113,6 +152,11 @@ public isolated function filter(map<Type> m, @isolatedParam function(Type val) r
 
 # Combines the members of a map using a combining function.
 #
+# ```
+# map<int> marks = {"Carl": 85, "Bob": 50, "Max": 60};
+# marks.reduce(isolated function (int accTot, int next) returns int => accTot + next, 0) ⇒ 195;
```

Comment:
As discussed in https://github.com/ballerina-platform/ballerina-lang/pull/39073/files#r1051801231